notebooks/image_models/solutions/diffusion_model.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>Can you give the necessary background to understand what's happening in the following line</p><p> </p><pre class="ql-syntax" spellcheck="false">noise_rates, signal_rates = diffusion_schedule(diffusion_times)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I think this should be more clear now with the added explanations + comments in the code. </p>